src/batchnorm/problem_description.cpp

```diff
@@ -178,6 +189,142 @@ NetworkConfig ProblemDescription::MakeNetworkConfig() const
     return NetworkConfig{ss.str()};
 }
 
+NetworkConfig ProblemDescription::MakeForwardInferenceNetworkConfig() const
+{
+    MIOPEN_THROW(miopenStatusNotImplemented);
+}
+
+NetworkConfig ProblemDescription::MakeBackwardNetworkConfig() const
+{
+    std::ostringstream ss;
+
+    bool bfpmixparm = false;
+    bool bfp16parm  = false;
+    bool bfp32parm  = true;
+    if(xDesc.GetType() == miopenHalf && GetScaleBiasDiffDesc().GetType() == miopenHalf)
+    {
+        bfp16parm = true;
+        bfp32parm = false;
+    }
+    else if(xDesc.GetType() == miopenHalf && GetScaleBiasDiffDesc().GetType() == miopenFloat)
+    {
+        bfpmixparm = true;
+        bfp32parm  = false;
+    }
+
+    int n, c, h, w;
+    std::tie(n, c, h, w) = tien<4>(xDesc.GetLengths());
+
+    const unsigned int in_cstride = h * w;
+    const unsigned int in_nhw     = n * in_cstride;
+
+    size_t xlocalsize = 1;
+    size_t ylocalsize = 1;
+
+    size_t xgridsize = 1;
+    size_t ygridsize = 1;
+
+    if(bn_mode == miopenBNSpatial)
```

Comment:
These computations (here and below) were moved from somewhere, but I don't see from where (no deleted similar code in this PR). Is it fine?

Reply:
This was copied from BatchnormBackward in batchnormocl.cpp and is required for values in the network config. They were not deleted there as they are also used in deciding which kernels to use and build parameters.

Reply:
Thanks, this matches my expectations, I just wanted to double-check.